source/includes/orr-intro.rst

```diff
@@ -0,0 +1,3 @@
+Starting in {+c2c-beta-program-short+} 1.8, you can enable Oplog Rollover 
+Resilience (ORR). With ORR,  ``mongosync`` applies changes made on the source 
+cluster to the destination cluster during the initial sync. 
```

Comment:
(short description)
I wonder if you could rework this last sentence to include a couple of words about why this is useful. You describe it really well in the reference page second paragraph. What do you think about adding some of that to this introduction.

Reply:
Noted and updated!